scripts/python/upload-with-submit.py

```
@@ -0,0 +1,239 @@
+    """Step 1: Initialize upload session."""
+    payload = {
+        "files": [{
+            "name": file_name,
+            "size": file_size,
+            "type": mime_type,
+            "lastModified": 1762656056765
+        }]
+    }
+
+    try:
+        r = requests.post(UPLOAD_INIT_URL, headers=COMMON_HEADERS, cookies=COOKIES, json=payload)
```

Ruff (0.14.4)

Probable use of `requests` call without timeout

(S113)